requeueFailedCRRCronJob.js

```diff
@@ -0,0 +1,284 @@
+const { waterfall, doWhilst, eachLimit, retry } = require('async');
+const http = require('http');
+const { Producer } = require('node-rdkafka');
+const { scheduleJob } = require('node-schedule');
+
+const { errors } = require('arsenal');
+const VID_SEP = require('arsenal').versioning.VersioningConstants
+      .VersionId.Separator;
+const { Logger } = require('werelogs');
+
+const BackbeatClient = require('./BackbeatClient');
+
+const log = new Logger('s3utils::requeueFailedCRRCronJob');
+const {
+    CLOUDSERVER_ENDPOINT,
+    BACKBEAT_API_ENDPOINT,
+    ACCESS_KEY,
+    SECRET_KEY,
+    SITE_NAME,
+    KAFKA_HOSTS,
+    KAFKA_TOPIC,
+} = process.env;
+const CRON_RULE = process.env.CRON_RULE || '0 */6 * * *';
+
+
+if (!CLOUDSERVER_ENDPOINT) {
+    throw new Error('CLOUDSERVER_ENDPOINT not defined');
+}
+if (!BACKBEAT_API_ENDPOINT) {
+    throw new Error('BACKBEAT_API_ENDPOINT not defined');
+}
+if (!ACCESS_KEY) {
+    throw new Error('ACCESS_KEY not defined');
+}
+if (!SECRET_KEY) {
+    throw new Error('SECRET_KEY not defined');
+}
+if (!SITE_NAME) {
+    throw new Error('missing SITE_NAME environment variable, must be set to' +
+                    ' the value of "site" property in the CRR configuration');
+}
+if (!KAFKA_HOSTS) {
+    throw new Error('KAFKA_HOSTS not defined');
+}
+if (!KAFKA_TOPIC) {
+    throw new Error('KAFKA_TOPIC not defined');
+}
+
+const PRODUCER_MESSAGE_MAX_BYTES = 5000020;
```

Comment:
are all the hardcoded values intentional?

Reply:
This is because the script is not integrated into backbeat (not yet at least), and it does not pull backbeat as a dependency, so we just redefine what are the values in the BackbeatProducer class for this purpose.